content/news/2003-09-27-installfest.md

```diff
@@ -0,0 +1,63 @@
+Title: InstallFest
+Tags: installfest
+Event: 2003-09-27 10:30 am to 4:30 pm
+Speaker: None
+Location: SSU Library, Room 1121
+Author: Dustin Mollo
+
+Details:
+
+Please note that this InstallFest is NOT at O'Reilly and Associates. It will instead be at Sonoma State University in Rohnert Park. Please see below for information on the exact location and directions.
+
+What is an InstallFest and who can come?
+
+Instead of putting all of the usual "what is an InstallFest" info in this and future announcements, we have a link up on our website as to what a NBLUG InstallFest entails and how to participate. If you point your browser of choice to http://www.nblug.org/installfest/, you'll be presented with all the info you ever wanted to know about what goes on at an InstallFest, and how you can get the most out of attending one.
```

Comment:
They automatically become clickable because Markdown, but yeah, *shrug*.  Don't worry about it.